docs/sources/http_api/short_url.md

```diff
@@ -0,0 +1,77 @@
++++
+title = "ShortURL HTTP API "
+description = "Grafana ShortURL HTTP API"
+keywords = ["grafana", "http", "documentation", "api", "shortUrl"]
+aliases = ["/docs/grafana/latest/http_api/short_url/"]
+type = "docs"
+[menu.docs]
+name = "ShortURL"
+parent = "http_api"
++++
+
+# ShortURL API
+
+## Redirect to full path from short URL UID
+
+`GET /api/goto/:uid`
+
+Redirects to the URL path associated with the given UID if it exists. Otherwise redirects to `/notfound`.
+
+**Example Request**:
```

Comment:
```suggestion
**Example request:**
```

Reply:
Update all your examples to match this format.

Reply:
thanks for all the help! looks like i picked some bad role models to emulate... have you considered proactively fixing some of these things everywhere, both for consistency and to ensure future contributors don't submit mistake-riddled documentation like i did? eg

![image](https://user-images.githubusercontent.com/5599894/93468141-8bba3d80-f8bc-11ea-9702-cf59e663677c.png)

vs

![image](https://user-images.githubusercontent.com/5599894/93468197-9d9be080-f8bc-11ea-815d-a95fbb0fd36e.png)

Reply:
We're working on it, but we have higher priorities. For example, writing and testing docs for the 7.2 release, rewriting older sections that are out-of-date, and creating new sections. We fix things as we touch them, but as you've noticed, there is a lot of content to go through.

Reply:
Btw, there's a #docs channel in the public Slack. I'm always up for chatting about docs. We could probably come up with some nice beginner doc issues to deal with some of these formatting issues.